spec/fortnox/api/repositories/customer_spec.rb

```diff
@@ -22,10 +19,11 @@
                    'test@example.com'
 
   # It is not yet possible to delete Customers. Therefore, expected nr of
-  # Customers when running .all will continue to increase.
+  # Customers when running .all will continue to increase
+  # (until 100, which is max by default).
   include_examples '.all', 100
```

Comment:
Good point!

Actually, our repository specs are run sequentially with the `order: :defined` in the spec declaration.

The tricky part is that there is a max limit of how many records Fortnox will return. Default is 100, which is what we have now. If we run `.all`, we will get 100 records back. Then we create a new one and run `.all` again and we will still get 100 records back.